mocha/test/version.test.js

```diff
@@ -0,0 +1,27 @@
+const httpStatus= require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+const testData = require("../utils/testData");
+
+let apiEndPoint= apiEndPoints.versionApiEndpoint;
+
+describe("Get Version endpoints test", function(){
+    it("user enters to see version",async function(){
+            let response= await apiHelper.sendPOSTRequest(
```

Comment:
@Selbinyyaz 
This is a GET endpoint so we don't use POST but sendGETRequest.
I think that if you rebase or merge from staging you will see this method.
this is the reason why your test doesn't work